docs/src/docs/asciidoc/developer/index.adoc

```diff
@@ -0,0 +1,33 @@
+= Ehcache 3 Developer Documentation
+== Code layout
+
+The code core of Ehcache is split in three core modules:
```

Comment:
core code?

Reply:
I just copied this section verbatim from the `Home.asciidoc` of the wiki.  Will fix however.